backend/src/govsg/services/govsg.service.ts

```diff
@@ -22,7 +22,13 @@ export async function getCampaignDetails(
       where: { campaignId },
       include: {
         model: GovsgTemplate,
-        attributes: ['id', 'body', 'params', 'param_metadata'],
+        attributes: [
+          'id',
+          'body',
+          'params',
+          'param_metadata',
+          'multilingualSupport',
```

Comment:
A bit disturbing that I have it camel case here. I'll check.

Reply:
`multilingualSupport` is correct. `param_metadata` came from `master` and I don't intend to risk breaking anything by making a commit on this.